src/test/java/com/revature/docutest/TestUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.revature.docutest;
+
+import io.swagger.models.Swagger;
+import io.swagger.parser.SwaggerParser;
+
+// container class for test data
+public class TestUtil {
+
+    public static Swagger todos;
+    public static Swagger blank;
+    public static Swagger malformed;
+    public static Swagger get;
+    public static Swagger multi;
+
+    {
+        initFields();
+    }
+
+    public static void initFields() {
```

Comment:
Is this style enforced by the rev_checks.xml? If not, it might be more clear to instantiate these fields at the top of the class, perhaps with a blank line between them for readability.